<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
border-radius:15px;
</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monotype Corsiva</family>
        <pointsize>29</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);
font: italic 29pt &quot;Monotype Corsiva&quot;;
text-decoration: underline;

</string>
      </property>
      <property name="text">
       <string>Add Student</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_first_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter First Name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_last_name">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter Last Name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_grd_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter Guardian's Name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_class">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter Class</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_phone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter Phone Number</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_grd_phone">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    Enter Guardian's Phone Number</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Guardian's Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Student Class:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Phone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Guardian's Phone:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_submit_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>470</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(0, 85, 255);

}

QPushButton:hover{
background-color: rgb(0, 143, 214);
	color: rgb(255, 255, 255);

}</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_dob">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>410</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgba(0, 0, 0, 0);
border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>    DD \ MM \ YYYY</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
</string>
      </property>
      <property name="text">
       <string>Date Of Birth:</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QRadioButton" name="add_female">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>6</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="add_male">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>6</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(0, 85, 255);

}

QPushButton:hover{
background-color: rgb(0, 143, 214);
	color: rgb(255, 255, 255);

}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Qt Desiger/Icon.qrc">
        <normaloff>:/icon/Icon/arrow-left-circle.svg</normaloff>:/icon/Icon/arrow-left-circle.svg</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="error">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 0, 0);
background:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>widget_2</zorder>
     <zorder>label_3</zorder>
     <zorder>add_first_name</zorder>
     <zorder>add_last_name</zorder>
     <zorder>add_grd_name</zorder>
     <zorder>add_class</zorder>
     <zorder>add_phone</zorder>
     <zorder>add_grd_phone</zorder>
     <zorder>label_20</zorder>
     <zorder>label_21</zorder>
     <zorder>label_22</zorder>
     <zorder>label_23</zorder>
     <zorder>label_24</zorder>
     <zorder>label_25</zorder>
     <zorder>add_submit_button</zorder>
     <zorder>add_dob</zorder>
     <zorder>label_26</zorder>
     <zorder>label_27</zorder>
     <zorder>back_button</zorder>
     <zorder>error</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Qt Desiger/Icon.qrc"/>
 </resources>
 <connections/>
</ui>
